packages/core/src/secret/index.ts

```
@@ -22,7 +23,7 @@ import {
   encryptStringAsync,
   ensureSensitiveTextEncoded,
 } from './encryptors/aes256';
-import { hash160 } from './hash';
+import { hash160, hash160Sync } from './hash';
```

ESLint

[error] hash160 not found in './hash'

(import/named)